.github/readmes/experimental/typescript/graphql-apollo-server/README.md

```diff
@@ -1,17 +1,17 @@
 # GraphQL Apollo Server Example
 
-This example shows how to implement a **GraphQL server with TypeScript** based on  [Prisma Client](https://github.com/prisma/prisma2/blob/master/docs/prisma-client-js/api.md), [apollo-server](https://www.apollographql.com/docs/apollo-server/) and [GraphQL Nexus](https://nexus.js.org/).
+This example shows how to implement a **GraphQL server with TypeScript** based on [Prisma Client](https://github.com/prisma/prisma2/blob/master/docs/prisma-client-js/api.md), [apollo-server](https://www.apollographql.com/docs/apollo-server/) and [Nexus Schema](https://nexusjs.org/#/components/schema/about).
 
-__INLINE(../../_setup-1.md)__
+**INLINE(../../\_setup-1.md)**
```

Comment:
not sure if this did not break the script. @nikolasburk

Reply:
Hm, prettier must have done that under my nose.

Reply:
It's back unfortunately.

Reply:
Is this a problem?

Reply:
(Happy to fix, just don't know if this matters)

Reply:
I just tested and it indeed breaks the "build script". By the way, to apply the changes you make to these "source READMEs" you need to do the following:

```
cd .github/tools
yarn
yarn build-readmes
``` 

This updates the _actual_ READMES in the `javascript`, `typescript` and `experimental` directories.